develop/toolkit/parachains/light-clients.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Light Clients
+description: Learn about Polkadot light clients, their importance, usage, and how they enable robust, cohesive, and secure application composition.
+---
+
+# Light Clients
+
+## Introduction
```

Comment:
This introduction is quite long. I'd scale it back to briefly introduce the topic, what problems it solves, what the guide covers using the usual recipe for intros